crowdsourcing/serializers/payment.py

```diff
@@ -0,0 +1,162 @@
+from crowdsourcing.models import Transaction, FinancialAccount, PayPalFlow, UserProfile
+from rest_framework import serializers
+from crowdsourcing.serializers.dynamic import DynamicFieldsModelSerializer
+from rest_framework.exceptions import ValidationError
+from django.contrib.auth.models import User
+from crowdsourcing.validators.utils import InequalityValidator, ConditionallyRequiredValidator
+from crowdsourcing.utils import PayPalBackend, get_model_or_none
+from rest_framework import status
+from django.utils import timezone
+
+class FinancialAccountSerializer(DynamicFieldsModelSerializer):
+    class Meta:
+        model = FinancialAccount
+        fields = ('id', 'owner', 'type', 'is_active', 'balance')
+
+
+class TransactionSerializer(DynamicFieldsModelSerializer):
+    class Meta:
+        model = Transaction
+        fields = ('id', 'sender_type', 'amount', 'currency', 'state', 'method',
+                  'sender', 'recipient', 'reference', 'created_timestamp', 'last_updated')
+        read_only_fields = ('created_timestamp', 'last_updated')
+
+    def create(self, *args, **kwargs):
+        transaction = Transaction.objects.create(**self.validated_data)
+        transaction.recipient.balance += transaction.amount
+        transaction.recipient.save()
+        if not transaction.sender.is_system:
```

Comment:
This will lead to massive inflation if we never reduce the system account's balance.